.github/actions/poetry/action.yaml

```diff
@@ -0,0 +1,36 @@
+name: 'Poetry Install'
```

Comment:
```suggestion
name: 'Build application'
```

Reply:
I think it is better to rename the subfolder to application and the the yaml to build.yaml